components/Episode/EpisodeCard.js

```diff
@@ -61,18 +66,25 @@ export default function EpisodeCard({
       ) : (
         <NoContentMessage>keine Beschreibung vorhanden</NoContentMessage>
       )}
+      {speakers?.length > 0 ? (
+        <Speakers speakers={speakers} />
+      ) : (
+        <NoContentMessage>keine Stimmen angegeben</NoContentMessage>
+      )}
       {chapters?.length > 0 ? (
         <Chapters chapters={chapters} />
       ) : (
         <NoContentMessage>
-          {parts ? "" : "keine Kapitelliste vorhanden"}
+          {parts?.length > 0 ? "" : "keine Kapitelliste vorhanden"}
         </NoContentMessage>
       )}
-      {parts?.length > 0
-        ? parts.map((part) => {
-            return <Parts key={part.buchstabe} part={part} />;
-          })
-        : ""}
+      {parts?.length > 0 &&
```

Comment:
```suggestion
      {parts?.length &&
```

Reply:
here it doesn't work with `&&` because `parts?.length > 0` returns a boolean while `parts?.length` seems to always return true in this case